waffle-compiler/src/compileNativeSolc.ts

```diff
@@ -14,7 +14,7 @@ export function compileNativeSolc(config: Config) {
 }
 
 export function createBuildCommand(config: Config) {
-  const command = 'solc';
+  const command = (config.compilerVersion === 'default') ? 'solc' : config.compilerVersion;
```

Comment:
I'm not a big typescript guru so not sure why this still defaults to `default` somehow, according to the tests.